src/app/core/audit/audit.service.js

```diff
@@ -23,28 +27,35 @@ const getMasqueradingUserDn = (eventActor, headers) => {
  * @param {string} message
  * @param {string} eventType
  * @param {string} eventAction
- * @param {* | Promise<*>} eventActor
- * @param {*} eventObject
- * @param {*} [eventMetadata]
- * @param {boolean} [stringifyEventObject]
- * @returns {Promise<import('./types').AuditDocument>}
+ * @param {import('express').Request | object} requestOrEventActor
+ * @param {object} eventObject
+ * @param {*} eventMetadata
+ * @returns {Promise<any>}
  */
 module.exports.audit = async (
 	message,
 	eventType,
 	eventAction,
-	eventActor,
+	requestOrEventActor,
 	eventObject,
-	eventMetadata = null,
-	stringifyEventObject = false
+	eventMetadata = null
 ) => {
 	/**
 	 * @type {import('./types').AuditModel}
 	 */
 	const Audit = dbs.admin.model('Audit');
-	const utilService = deps.utilService;
 
-	const actor = await Promise.resolve(eventActor);
```

Comment:
If this still comes in as a promise, we should still await it, no?

Reply:
Misread the comment annotation. I see we're no longer accepting a promise here. Good to know.

Reply:
I removed it, because I was certain that we didn't ever pass in a promise.  Double checked and to my surprise,`TeamMember.auditCopy` returns a promise, so we always pass in a promise.  Just pushed an update.